sonic_sfp/sff8436.py

```diff
@@ -364,6 +364,41 @@ class sff8436InterfaceId(sffbase):
                  'type'  : 'bitmap',
                  'decode': {}}}
 
+    sfp_type = {
+        'type':
+            {'offset':0,
```

Comment:
Add space after `:` for consistent PEP8 style (`{'offset': 0,`).

There are many more occurrences. Please do this throughout the entire file.